bench.go

```diff
@@ -60,7 +62,7 @@ func (s *scaleOut) run() error {
 	}
 	s.t.addTime = time.Now()
 	for {
-		time.Sleep(time.Minute)
+		//time.Sleep(time.Minute)
```

Comment:
We should keep it here to avoid that the state is still balanced just after the scale out